src/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -4487,66 +4487,6 @@ elementFormDefault="qualified">
-    <xs:element name="_SymbolPackageToBundle" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="_TestLayoutSourceFiles" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="_TestLayoutTargetFiles" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="_UnfilteredAppxPackagePayload" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="_UnfilteredRecursiveResolvedSDKReference" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="_WinmdFilesFromOtherGroups" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="_WinmdFilesFromReferences" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="_WinmdFilesFromSDKs" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="_WinmdFilesFromWinmdArtifacts" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="_WinmdWithImplementation" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="AppxBundlePlatformWithAnyCPU" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="AppxManifestForBundle" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="CustomAppxManifest" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="FileReads" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="FileWrites" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="FinalAppxManifest" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="FinalAppxPackageItem" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="FinalAppxSymbolPackageItem" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="FrameworkSdkReference" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="NonFrameworkSdkReference" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="PackagingDirectoryWrites" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="PackagingFileWrites" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="PackagingOutputs" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="PDBPayload" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="ProjectArchitecture" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="SourceAppxManifest" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-    <xs:element name="UpToDateCheckOutput" type="msb:SimpleItemType" substitutionGroup="msb:Item" />
-
     <xs:element name="AppxPackagePayload" substitutionGroup="msb:Item">
         <xs:complexType>
             <xs:complexContent>
```

Comment:
There are several things here that I think are logically under that comment--there's no doc strings, so the autocompletion isn't great--but they do describe structure, so I'm leaving them. Should we delete everything that doesn't have doc strings?